api/v1alpha3/machine_types.go

```diff
@@ -29,6 +29,9 @@ const (
 	// MachineControlPlaneLabelName is the label set on machines or related objects that are part of a control plane.
 	MachineControlPlaneLabelName = "cluster.x-k8s.io/control-plane"
 
+	// MachineUnhealthy is set on machines determined unhealthy by the Machine Health Check controller.
```

Comment:
The comment should be explicit about the contract, e.g. expect a machine to be deleted by the owning controller, etc.

Reply:
Added another line, is this what you had in mind?

Reply:
Looks like the extra comment line may have been dropped?

Reply:
Indeed, somehow messed something up in rebasing it seems